spec/acceptance/mod_suphp_spec.rb

```diff
@@ -1,6 +1,10 @@
 require 'spec_helper_acceptance'
 
-describe 'apache::mod::suphp class', if: (fact('operatingsystem') == 'Ubuntu' && fact('operatingsystemmajrelease') != '16.04') do
+# As of 2013 suphp was declared EOL by it's creators and as such support for it has been removed from Ubunto 15.10/Debian 8 onward.
+# https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=738133
+# https://www.howtoforge.com/tutorial/perfect-server-debian-8-jessie-apache-bind-dovecot-ispconfig-3/2/#-install-suphp-optional-not-recommended
+describe 'apache::mod::suphp class', if:  (fact('operatingsystem') == 'Ubuntu' && (fact('operatingsystemmajrelease') < '15.10')) ||
```

Comment:
These distros aren't listed in metadata.json so I wonder if it's even relevant to keep this. On the other hand, the unit tests are still run on RH so you might switch it to that.

Reply:
Was hesitant on outright removing, but what the hell.